### ISSUE TITLE
Logging rules not flagged if logger imported from another module

### ISSUE THREAD

**Issue by petermattia** [astral-sh/ruff#5694]
I'm observing that logging rules aren't flagged if a logger is imported from another module.

* A minimal code snippet that reproduces the bug.

If I set up logging here, both of these violations are flagged:
````python
# logging_setup.py
import logging

linter = "ruff"
logging.info(f"Hello, {linter}!") # G004 flagged

logger = logging.getLogger(__name__)

logger.info(f"Hello, {linter}!") # G004 flagged
````

However, if I import `logger` into another module, the below identical violation is not flagged:
````python
# my_module.py
from logging_setup import logger

linter = "ruff"
logger.info(f"Hello, {linter}!") # G004 *not* flagged
````

* The command you invoked (e.g., `ruff /path/to/file.py --fix`), ideally including the `--isolated` flag.

`ruff .`

* The current Ruff settings (any relevant sections from your `pyproject.toml`).

`"G"` ruleset

* The current Ruff version (`ruff --version`).
`0.0.277`

**Comment by petermattia** [comment]
Hi `ruff` team, friendly follow up on this one, please let me know if I'm missing something here. Thanks!

**Comment by charliermarsh** [comment]
Will take a look.

**Comment by charliermarsh** [comment]
Looks like an oversight, we would flag `logging_setup.logger.info(f"Hello, {linter}!")` but not the `import from` variant.

**Comment by petermattia** [comment]
Thanks @charliermarsh, that makes sense.

**Comment by charliermarsh** [comment]
I think I misdiagnosed this, `logging_setup.logger.info` also isn't working in my initial testing, but regardless it'll be fixed in the next release.
